setup.py

```diff
@@ -363,10 +364,11 @@ def check_file_at_path(path2file):
 
 packages = find_packages(include=["numba", "numba.*"])
 
-build_requires = ['numpy >={}'.format(min_numpy_build_version)]
+build_requires = ['numpy >={},{}'.format(min_numpy_build_version,
```

Comment:
```suggestion
build_requires = ['numpy >={},<{}'.format(min_numpy_build_version,
```

This is missing a less than symbol causing RTD to fail.

Reply:
fixed